harness/determined/pytorch/deepspeed/_deepspeed_trial.py

```diff
@@ -646,27 +1031,38 @@ def _load(self, load_path: pathlib.Path) -> None:
                     "callback will be initialized from scratch"
                 )
 
-        # Load workload sequencer state.
-        wlsq_path = load_path.joinpath("workload_sequencer.pkl")
```

Comment:
you need this. should test the case of loading a pre-deepspeed-trainer checkpoint into a post-deepspeed-trainer world

Reply:
you resolved this but i still don't see the legacy workload seq checkpoint being loaded?